README.md

```diff
@@ -27,3 +27,11 @@ This action slug and expose some github variables.
     echo ${{ env.GITHUB_HEAD_REF_SLUG }}
     echo ${{ env.GITHUB_BASE_REF_SLUG }}
 ```
+
+# Test
+
+To run test locally, run:
+
+```shell script
+docker run -w /workdir -v $(pwd):/workdir dduportal/bats:latest ./
```

Comment:
Can you put this command line under a `test` task in a `Makefile` file?

```
.PHONY: tests
tests:
	docker run -w /workdir -v $(shell pwd):/workdir dduportal/bats:latest ./
```

So we can just run `make tests`

Reply:
done